pkg/aws/aws.go

```diff
@@ -170,3 +188,28 @@ func (a *AwsClient) listInstancesWithFilter(filters []*ec2.Filter) ([]*ec2.Insta
 	}
 	return instances, nil
 }
+
+// ListStopInstancesEvents lists StopInstances events from CloudTrail
+func (a AwsClient) ListStopInstancesEvents() ([]*cloudtrail.Event, error) {
```

Comment:
It may make sense to define this function differently. At the moment we are able to list all events of all instances. However, I think we rather need to list events on stopped instances meaning:

```suggestion
func (a AwsClient) ListStopInstancesEvents(instanceID string) ([]*cloudtrail.Event, error) {
```

Reply:
however this strongly depends on what we are returning [here](https://github.com/openshift/configuration-anomaly-detection/blob/739c2b02d9c4d38bb0647b9b3a6b5413595783ea/pkg/aws/aws.go#L126)